<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveWidget</class>
 <widget class="QWidget" name="SaveWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>59</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>输出栅格路径</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="mSaveEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="mSaveBtn">
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
